src/test/regress/sql/multi_explain.sql

```diff
@@ -17,11 +41,21 @@ EXPLAIN (COSTS FALSE, FORMAT JSON)
 	SELECT l_quantity, count(*) count_quantity FROM lineitem
 	GROUP BY l_quantity ORDER BY count_quantity, l_quantity;
 
+-- Validate JSON format
```

Comment:
I really liked the idea of `explain_json` and `explain_xml` functions.  I see tests for real-time and task-tracker executors with the functions. Do you think is it worth to add a router executor as well?